test/utils/wabt_engine.cpp

```diff
@@ -0,0 +1,110 @@
+#include "src/binary-reader.h"
+#include "src/cast.h"
+#include "src/feature.h"
+#include "src/interp/binary-reader-interp.h"
+#include "src/interp/interp.h"
+#include "src/literal.h"
+#include "src/option-parser.h"
+#include "src/resolve-names.h"
+#include "src/stream.h"
+#include "src/validator.h"
+#include "src/wast-lexer.h"
+#include "src/wast-parser.h"
```

Comment:
Any way to fix these and use something like `<wabt/`> ?

Reply:
Only by some workarounds. These headers include others this way.